packages/web-forms/e2e/vue.test.ts

```diff
@@ -76,7 +76,7 @@ test('All forms are rendered and there is no console error', async ({ page, brow
 		const langChanger = page.getByLabel('change language');
 
 		if ((await langChanger.count()) > 0) {
-			await langChanger.click();
+			await langChanger.first().click();
```

Comment:
This feels like something that could churn quite a lot. I'm generally not fond of littering UI with classes just for testing purposes, but it seems reasonable to make an exception for:

- things that are global singletons (like this is)

- E2E testing specifically, where it's already challenging to relate the test code to the intended structure it's targeting

Reply:
Added classes to make it easier to select language changer in e2e test.